### ISSUE TITLE
"SyntaxError: Unexpected token )" on Node v6.0.0

### ISSUE THREAD

**Issue by oles** [keithwhor/nodal#222]
```
/home/oles/project/node_modules/nodal/core/required/model.js:1016
      callback = callback || () => {};
                              ^
SyntaxError: Unexpected token )
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:511:25)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at /home/oles/project/node_modules/nodal/core/required/api.js:5:17
    at Object.<anonymous> (/home/oles/project/node_modules/nodal/core/required/api.js:143:3)
```

Happens when running "node cluster.js" - and started happening when I updated to Node 6  :sweat_smile:

**Comment by keithwhor** [comment]
Yeah --- if you can run Nodal on <6 for now, I'll have a 6.0 update (Nodal 0.10) by Monday for you. :) Lots of stuff changing around.

**Comment by keithwhor** [comment]
I'll leave this open for now :)

**Comment by oles** [comment]
Not a problem, rolled back to v5.11.0 - and it worked again :smile: 

Love your work on Nodal <3

**Comment by nsipplswezey** [comment]
I think this can be closed since 0.9.1 hotfixes were released.

**Comment by oles** [comment]
Indeed it can, working as it should at v6.1.0
